pkg/console/install_panels.go

```diff
@@ -547,6 +550,34 @@ func addDiskPanel(c *Console) error {
 		}
 		return showNext(c, persistentSizePanel)
 	}
+
+	// isWipeDisksNeeded is a helper function to render the wipeDisksPanel if needed
+	// if there are no additional disks to wipe then it checks if MBR needs to be enabled
+	// else will move on to the next apge
+	isWipeDisksNeeded := func(g *gocui.Gui, v *gocui.View) error {
+		options, err := getWipeDisksOptions(c.config)
```

Comment:
Is there an option for the user to not select a disk, and leave this blank? Or is it required to always provide at least one disk to wipe?